src/coreclr/gc/gc.cpp

```diff
@@ -43992,6 +44031,92 @@ class NoGCRegionLockHolder
     }
 };
 
+EnableNoGCRegionCallbackStatus gc_heap::enable_no_gc_callback(uint64_t total_size)
```

Comment:
>uint64_t total_size) [](http://example.com/codeflow?start=62&length=20)

perhaps change this to callback_threshold? I'm not sure what to make of total_size.